packages/util/src/api.ts

```diff
@@ -305,22 +305,37 @@ export class Api extends BaseApi {
    * Import account by PolkadotJs extension
    * @param address
    * @param name
+   * @param isDesktop
    */
-  public importByPolkadotJs(address: string, name: string): void {
-    const account = keyring.addExternal(address, { name: name || '' });
+  public importByPolkadotJs(address: string, name: string, isDesktop = false): void {
+    let account;
+    if (isDesktop) {
+      const pair = keyring.getPair(address);
+      account = { pair };
+    } else {
+      account = keyring.addExternal(address, { name: name || '' });
+    }
 
     this.setAccount(account);
 
     if (this.storage) {
       const soraAddress = this.formatAddress(account.pair.address);
       this.storage.set('name', name);
       this.storage.set('address', soraAddress);
-      this.storage.set('isExternal', true);
+      this.storage.set('isExternal', !isDesktop);
     }
 
     this.initAccountStorage();
   }
 
+   /**
+   * Unlock account to sign tx
+   * @param password
+   */
+  public unlockPair(password: string): void {
```

Comment:
Is it necessary to define this method in `Api `class? as `Api ` extends from `BaseApi`, you should be able to use inherited method like api.inheritedMethod

Reply:
where `inheritedMethod()` is in BaseApi.ts? Ok will fix that